### PR TITLE
chore(mysql): add missing error check in example

### PR DESCRIPTION
**Pull request by stevenh**
Add a missing error check in the mysql example for connection setup.

### CODE REVIEW

File: modules/mysql/examples_test.go [hunk 1]
```diff
@@ -66,7 +66,10 @@ func ExampleRun_connect() {
 		}
 	}()
 
-	connectionString, _ := mysqlContainer.ConnectionString(ctx)
+	connectionString, err := mysqlContainer.ConnectionString(ctx)
+	if err != nil {
+		log.Fatalf("failed to get connection string: %s", err)
```
Comment (mdelapenya) [inline modules/mysql/examples_test.go:71]:
```suggestion
		log.Fatalf("failed to get connection string: %s", err) // nolint:gocritic
```

Comment (stevenh) [reply modules/mysql/examples_test.go:71]: FYI: This gets reworked in a latter PR as all the examples are broken

Comment (mdelapenya) [reply modules/mysql/examples_test.go]: Broken in what manner? Is the lint complaining?

Comment (stevenh) [reply modules/mysql/examples_test.go]: If an error occurs clean up doesn't happen as `log.Fatalf` issues an `os.Exit(1)` so the defers don't run.

Comment (mdelapenya) [reply modules/mysql/examples_test.go]: Ah oh yes, sure, thanks for the explanation. You're very right 🙇

**Pull request merged**